Makefile

```diff
@@ -11,3 +11,5 @@ clean:
 	rm -f $(OBJS)
 	rm -f gpio-watch
 
+install:
+	sudo install -m 664 gpio-watch /usr/bin
```

Comment:
Setting mode 644 would make the resulting file not executable. You would want mode 0755, probably.  I also dislike having `sudo` commands hidden inside Makefiles; I might intentionally be running things as  non-root user to avoid unexpected changes to the filesystem.